iphone/Maps/Classes/MapViewController.mm

```diff
@@ -579,22 +579,19 @@ - (void)processMyPositionStateModeEvent:(location::EMyPositionMode)mode
 
   switch (mode)
   {
-    case location::MODE_UNKNOWN_POSITION:
-    {
+    case location::PendingPosition:
       self.disableStandbyOnLocationStateMode = NO;
-      if (![Alohalytics isFirstSession])
-        [[MapsAppDelegate theApp].locationManager stop:self];
+      [[MapsAppDelegate theApp].locationManager start:self];
       break;
-    }
-    case location::MODE_PENDING_POSITION:
+    case location::NotFollowNoPosition:
+      //TODO(iOS team): show dialog if it is not first launch
```

Comment:
А можно конкретнее?

> On 14.4.2016, at 11.35, Roman Kuznetsov <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/MapViewController.mm:
> 
> >        break;
> > -    }
> > -    case location::MODE_PENDING_POSITION:
> > +    case location::NotFollowNoPosition:
> > +      //TODO(iOS team): show dialog if it is not first launch
> iOS команда знает о каком
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>